deploy.sh

```diff
@@ -2,32 +2,25 @@
 
 set -e # exit with nonzero exit code if anything fails
 
-echo 'Travis Build Dir: '$TRAVIS_BUILD_DIR
-
 if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
Any reason why this is `!= "false" ` instead of just `== "true" ` ?

Reply:
This environment variable will contain the pull request number if it is a pull request and false if not.